tests/richenum/test_rich_enums.py

```diff
@@ -147,3 +147,18 @@ def test_unicode_handling(self):
         self.assertEqual(str(poop_okra), "Okra💩")
         if not PY3:
             self.assertEqual(unicode(poop_okra), u"Okra💩")
+
+    def test_specific_lookup_error_is_caught(self):
+        with self.assertRaises(Vegetable.LookupError):
+            Vegetable.lookup('canonical_name', 'meat')
+
+    def test_other_specific_lookup_error_is_not_caught(self):
+        class Meat(RichEnum):
+            COW = RichEnumValue("cow", "Cow")
+
+        try:
```

Comment:
@dhui I thought about that, but what I'm actually trying to assert here is that catching `Meat.LookupError` won't catch `<NotMeat>.LookupError`, so I think this expresses the intent?